kratos/python_scripts/analysis_stage.py

```diff
@@ -60,12 +60,29 @@ def RunSolutionLoop(self):
         """This function executes the solution loop of the AnalysisStage
         It can be overridden by derived classes
         """
+        # We check of some model part requires reinitialize
+        modified_model = False
+        if hasattr(self, 'model'):
+            model_part_names = self.model.GetModelPartNames()
+            for model_part_name in model_part_names:
+                if self.model.GetModelPart(model_part_name).Is(KratosMultiphysics.MODIFIED):
```

Comment:
Should the tag name not be 'MODIFIABLE'?. Otherwise the natural interpretation is of something that may or may not change at each time step.

Reply:
The proper thing woul be to create a new flag, but requires a whole discussion